server/src/main/java/org/opensearch/rest/action/cat/RestMasterAction.java

```diff
@@ -50,18 +50,19 @@
 public class RestMasterAction extends AbstractCatAction {
```

Comment:
Such class name won't be considered as inclusive?

Reply:
Hi @owaiskazi19, thanks for your review! The `RestMasterAction` class is published to maven as Java API (https://opensearch.org/javadocs/1.2.4/OpenSearch/server/build/docs/javadoc/org/opensearch/rest/action/cat/RestMasterAction.html). I didn't plan to rename them in 2.0 version, in case giving clients or plugins which use such classes too much work in upgrading to support 2.0 version. 😄 
There is a discussion in the PR https://github.com/opensearch-project/OpenSearch/pull/2453, you could take a look at. Renaming the Java APIs is tracked in issue https://github.com/opensearch-project/OpenSearch/issues/1684

Reply:
Sounds good. Thanks for clearing it.

Reply:
You are welcome! 😁